packages/nodes-base/nodes/Telegram/Telegram.node.ts

```diff
@@ -208,6 +208,12 @@ export class Telegram implements INodeType {
 						description: 'Delete a chat message',
 						action: 'Delete a chat message',
 					},
+					{
+						name: 'Edit Message Caption',
+						value: 'editMessageCaption',
+						description: 'Edit a caption of message',
+						action: 'Edit a caption of test message',
```

Comment:
I think the Description and action might be better as

"Edit the caption of a message"

I am unable to make this change as you have used the master branch not a standard branch, if you can update this I can fit it in for a second review on Monday or next Thursday.